content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -0,0 +1,97 @@
+  text: "AWS Integration"
+- link: "https://docs.datadoghq.com/serverless/libraries_integrations/forwarder/"
+  tag: "Documentation"
+  text: "Datadog Forwarder Lambda function"
+- link: "https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination/"
+  tag: "Guide"
+  text: "Send AWS service logs with the Datadog Kinesis Firehose destination"
+- link: "https://docs.datadoghq.com/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/"
+  tag: "Guide"
+  text: "AWS CloudWatch metric streams with Kinesis Data Firehose"
+- link: "https://www.datadoghq.com/blog/aws-monitoring/"
+  tag: "Blog"
+  text: "Key metrics for AWS monitoring"
+- link: "https://www.datadoghq.com/blog/monitoring-ec2-instances-with-datadog/"
+  tag: "Blog"
+  text: "How to monitor EC2 instances with Datadog"
+- link: "https://www.datadoghq.com/blog/monitoring-aws-lambda-with-datadog/"
+  tag: "Blog"
+  text: "Monitoring AWS Lambda with Datadog"
+- link: "https://www.datadoghq.com/blog/cloud-security-posture-management/"
+  tag: "Blog"
+  text: "Introducing Datadog Cloud Security Posture Management"
+- link: "https://www.datadoghq.com/blog/datadog-workload-security/"
+  tag: "Blog"
+  text: "Secure your infrastructure in real time with Datadog Cloud Workload Security"
+- link: "https://www.datadoghq.com/blog/announcing-cloud-siem/"
+  tag: "Blog"
+  text: "Announcing Datadog Security Monitoring"
+---
+
+## Overview
+
+Use this guide to manually set up the Datadog [AWS Integration][1].
+
+Setting up the AWS integration manually involves creating an IAM policy and IAM role in your AWS account, and configuring the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.
+
+## Setup
+
+### Generate an external ID
+Generate an External ID in the <a href="https://app.datadoghq.com/account/settings#integrations/amazon_web_services" target="_blank">AWS integration tile</a> on the integrations page. This is used in the trust policy of the AWS IAM role you create for Datadog.
+1. Select the `Configuration` tab, then `Role Delegation`.
+2. Click `Manually`. This creates an AWS External ID which is used for configuration of the AWS IAM role. For more information about the External ID, see the [IAM User Guide][2].
+3. Copy this value to your clipboard or notepad.  
+  **Note: Do not close the integration tile or the Datadog application page**, as this causes the external ID value to reset.
+
+### AWS IAM Policy for Datadog
+Create an IAM policy for the Datadog role in your AWS account with [all permissions](#datadog-permissions) to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change. 
```

Comment:
```suggestion
Create an IAM policy for the Datadog role in your AWS account with the [necessary permissions](#datadog-permissions) to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change. 
```

Reply:
@buraizu also this should link to the the AWS Integration IAM Policy, not just the permissions section
https://docs-staging.datadoghq.com/bryce/create-aws-manual-setup-guide/integrations/guide/aws-manual-setup/#aws-integration-iam-policy